std_msgs/Header header
float32 motor_temp      # motor temperature
